txscript/standard.go

```diff
@@ -44,23 +44,27 @@ type ScriptClass byte
 
 // Classes of script payment known about in the blockchain.
 const (
-	NonStandardTy ScriptClass = iota // None of the recognized forms.
-	PubKeyTy                         // Pay pubkey.
-	PubKeyHashTy                     // Pay pubkey hash.
-	ScriptHashTy                     // Pay to script hash.
-	MultiSigTy                       // Multi signature.
-	NullDataTy                       // Empty data-only (provably prunable).
+	NonStandardTy       ScriptClass = iota // None of the recognized forms.
+	PubKeyTy                               // Pay pubkey.
+	PubKeyHashTy                           // Pay pubkey hash.
+	WitnessPubKeyHashTy                    // Pay witness pubkey hash.
+	ScriptHashTy                           // Pay to script hash.
+	WitnessScriptHashTy                    // Pay to witness script hash.
+	MultiSigTy                             // Multi signature.
+	NullDataTy                             // Empty data-only (provably prunable).
 )
 
 // scriptClassToName houses the human-readable strings which describe each
 // script class.
 var scriptClassToName = []string{
-	NonStandardTy: "nonstandard",
-	PubKeyTy:      "pubkey",
-	PubKeyHashTy:  "pubkeyhash",
-	ScriptHashTy:  "scripthash",
-	MultiSigTy:    "multisig",
-	NullDataTy:    "nulldata",
+	NonStandardTy:       "nonstandard",
+	PubKeyTy:            "pubkey",
+	PubKeyHashTy:        "pubkeyhash",
+	WitnessPubKeyHashTy: "witnesspubkeyhash",
```

Comment:
This should be `witness_v0_keyhash` to match Core's output.  There are probably applications which rely on looking at the script type via RPC.